### PR TITLE
[16.0][mig] project_timesheet_time_control

### PR DESCRIPTION
**Pull request by aktiv-heli-kantawala**

### CODE REVIEW

**Comment by DorianMAG** [comment]
Hi @heliaktiv,
What branch have you use to resolv dependencies "web".
I find this for web_ir_actions_act_multi [2328](https://github.com/OCA/web/pull/2328)
I cant find for web_ir_actions_act_view_reload
Regards

File: project_timesheet_time_control/tests/test_project_timesheet_time_control.py [hunk 1]
```diff
@@ -145,8 +144,6 @@ def test_aal_without_start_resume_button(self):
         """If a line has no start date, can only resume it."""
         self.assertEqual(self.line.show_time_control, "stop")
         self.line.date_time = False
-        self.line.invalidate_cache()
-        self.assertEqual(self.line.show_time_control, "resume")
```
Comment (stefan-tecnativa) [inline project_timesheet_time_control/tests/test_project_timesheet_time_control.py]: Why are you removing this checks?

**Comment by stefan-tecnativa** [comment]
You can remove the dependencies of `web_ir_actions_act_multi` and `web_ir_actions_act_view_reload`. I've tested with @CarlosRoca13 that the module maintains it's functionallity without this dependencies.
ping @DorianMAG

**Comment by DorianMAG** [comment]
Hi @stefan-tecnativa .
Thx for this information.
I will test that
Regards

**Comment by DorianMAG** [comment]
Hi.
Some dependencies are almost merged.
https://github.com/OCA/timesheet/pull/586
https://github.com/OCA/timesheet/pull/592
i will try after this merges

**Comment by AntoniRomera** [comment]
@heliaktiv Dependencies have already been merged

**Comment by pedrobaeza** [comment]
Superseded by #1148

**Pull request closed without merging**